Assets/Mirage/Runtime/ClientObjectManager.cs

```diff
@@ -58,12 +63,28 @@ public void Start()
 
                 if (NetworkSceneManager != null)
                     NetworkSceneManager.OnClientFinishedSceneChange.AddListener(OnFinishedSceneChange);
+
+                if (ObjectPooling)
+                {
+                    _objectPoolingManager = new ObjectPoolingManager(null, gameObject);
+                }
             }
         }
 
         void OnClientConnected(INetworkPlayer player)
         {
             syncVarReceiver = new SyncVarReceiver(Client, Client.World);
+
+            if (ObjectPooling)
+            {
+                for (int i = 0; i < spawnPrefabs.Count; i++)
```

Comment:
can you use a foreach loop here? it'll be easier to read :)

Reply:
I will do a stopwatch test and if its not noticeable difference then np. I will post the test in a few

Reply:
done